frontends/concrete-python/benchmarks/levenshtein_distance.py

```diff
@@ -0,0 +1,157 @@
+"""
+Benchmarks of the levenshtein distance example.
+"""
+
+from functools import lru_cache
+from pathlib import Path
+from typing import Tuple
+
+import py_progress_tracker as progress
+
+from concrete import fhe
+from examples.levenshtein_distance.levenshtein_distance import Alphabet, LevenshteinDistance
+
+
+@lru_cache
```

Comment:
why don't we import this function from the main examples file? here, we copy so it's going to diverge

Reply:
It's not the same logic. The one in the examples takes the full module and does encryption in it's body.
This one just executes already encrypted inputs in a server, without the full module, or even a client (we're just using the evaluation keys from the client, maybe I can make it the parameter directly).

Reply:
- Examples are for showing how to make interesting tasks in FHE. They are focused on the definition of the circuit/module.
- Tests are for validating whether the implementations work as intended. They are focused on the semantics of the circuit/module.
- Benchmarks are for understanding the performance characteristics of the implementation. They are focused on the execution time of the circuit/module.

So yes, since their purpose is distinct, we can have 2 different evaluation paths.

I can rename the function to `levenshtein_on_server`, to signify it's purpose!

Reply:
yes thx!